### ISSUE TITLE
Retire Sims

### ISSUE THREAD

**Issue by tomascw** [mysimulation/NewSimsOnline#14]
This is becoming a highly requested feature. We need to look into making this happen. I've figured we can look into how IP banning works and see how can we reverse it's functionality for retiring Sims.

**Comment by jackmawer** [comment]
It's important to look into doing this manually ASAP in the mean time - does deleting DB entries do this?

**Comment by LetsRaceBwoi** [comment]
Yes.  Deleting the DB entry for an avatar will _technically_ retire it.
